tests/helixperftasks.targets

```diff
@@ -15,8 +15,8 @@
     </GetPerfTestAssemblies>
 
     <PropertyGroup>
-      <TestRuntimeDependenciesJson>$(SourceDir)Common\test_runtime\project.json</TestRuntimeDependenciesJson>
-      <TestRuntimeProjectLockJson>$(SourceDir)Common\test_runtime\project.lock.json</TestRuntimeProjectLockJson>
+      <TestRuntimeDependenciesJson>$(TestRuntimeProjectJson)</TestRuntimeDependenciesJson>
```

Comment:
nit: I think we can simply replace all references to TestRuntimeDependenciesJson and TestRuntimeDependenciesJson with $(TestRuntimeProjectJson) and $(TestRuntimeProjectLockJson) respectively for this particular file.